ui/pages/confirmation/templates/flask/snap-confirm/snap-confirm.js

```diff
@@ -45,7 +45,11 @@ function getValues(pendingApproval, t, actions) {
               children: {
                 element: 'TextArea',
                 props: {
-                  height: '400px',
+                  // TODO(ritave): Terrible hard-coded height hack. Fixing this to adjust automatically to current window height would
+                  //               mean allowing template compoments to change global css, and since the intended use of the template
+                  //               renderer was to allow users to build their own UIs, this would be a big no-no. We probably should
+                  //               deprecate the renderer altogether anyways
+                  height: '238px',
```

Comment:
```suggestion
                  // TODO(ritave): Terrible hard-coded height hack. Fixing this to adjust automatically to current window height would
                  //               mean allowing template compoments to change global css, and since the intended use of the template
                  //               renderer was to allow users to build their own UIs, this would be a big no-no.
                  height: '238px',
```